### PR TITLE
Add governance

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

As @Phyloiota requested I imported the [Governance](https://github.com/iota-community/Shimmer-Governance) content with its full history.

TODO:

- [x] Add proper frontmatter
- [x] Let Governance team check the docs as small changes had to be done
- [x] Fix links
- [x] Add to next section

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: shimmer/learn/governance/the-shimmer-governance-framework.md [hunk 1]
```diff
@@ -0,0 +1,933 @@
...
+title: The Shimmer Governance Framework
+description: This is the full specification for all governance processes in the Shimmer network. Detailed rules and procedures are defined here.
+image: /img/learn/governance/shimmer-gov.png
+keywords:
+  - Specification
+  - Shimmer
+  - Governance
+  - Framework
+---
+# The Shimmer Governance Framework
```
Comment (Dr-Electron) [inline shimmer/learn/governance/the-shimmer-governance-framework.md:11]: This is not needed. We already have the title in the frontmatter


```suggestion
```

**Review by Dr-Electron — COMMENTED** [review]
Looks "good enough" to me :trollface: 
The governance team may think about rewriting some parts imo as it's quite hard to read all of this. For example by adding [notes](https://docusaurus.io/docs/markdown-features/admonitions), using consisting formatting in lists, shorten paragraphs and so on

**Comment by Dr-Electron** [comment]
@lucas-tortora @jlvandenhout if you want to give it a quick read. But we won't do bigger changes anymore. So if you want to change something that can not done by suggestion, please create another PR. This one will be merged tonight.

**Review by Phyloiota — APPROVED** [review]
Agree wit @Dr-Electron - it can and will be updated in the future, but all things written in the specification must be seen as "law" currently and so the wordings are particular difficult to change. Also all changes will need to be ratified by community votes in the future which are planned to happen quarterly.
Important please for everyone that no changes to these documents should be approved without requesting reviews by me in the future (as i will have to bring it to the governance group and potentially have to fascilate a proposal for those changes)

**Comment by lucas-tortora** [comment]
> @lucas-tortora @jlvandenhout if you want to give it a quick read (you don't have to). But we won't do bigger changes anymore. So if you want to change something that can not done by suggestion, please create another PR. This one will be merged tonight.

I doubt I'll have time to go into depth today. So if you don't mind, we can add it to our board, and I can create a PR for style and structure next week. WDYT?

**Pull request merged**